jupyterhub/tests/test_api.py

```diff
@@ -966,6 +966,42 @@ def test_groups_list(app):
     }]
 
 
+@mark.group
+@mark.gen_test
+def test_add_multi_group(app):
+    db = app.db
+    names = ['group1', 'group2']
+    r = yield api_request(app, 'groups', method='post',
+                          data=json.dumps({'groups': names}),
+                          )
+    assert r.status_code == 201
+    reply = r.json()
+    r_names = [group['name'] for group in reply]
+    assert names == r_names
+
+    # for name in names:
+    #     user = find_user(db, name)
+    #     assert user is not None
+    #     assert user.name == name
+    #     assert not user.admin
```

Comment:
Do you want to restore or delete these commented blocks?

Reply:
Oh sorry. I will delete those comments.